packages/babel-core/src/transformation/file/options/build-config-chain.js

```diff
@@ -61,8 +61,16 @@ class ConfigChainBuilder {
       if (!foundConfig) {
         let configLoc = path.join(loc, BABELRC_FILENAME);
         if (exists(configLoc)) {
-          this.addConfig(configLoc);
-          foundConfig = true;
+          foundConfig = this.addConfig(configLoc);
+        }
+
+        let configJSLoc = path.join(loc, BABELRC_JS_FILENAME);
+        if (exists(configJSLoc)) {
+          if (foundConfig) {
+            throw new Error(`Error while loading config: multiple files found. Please remove one:\n- ${configLoc}\n- ${configJSLoc}`);
+          } else {
+            foundConfig = this.addConfig(configJSLoc);
```

Comment:
Agreed that thI current checks aren't the most intuitive. Do you think we should still throw an error when both a .babelrc and a .babelrc.js file are found?